src/NexusMods.SingleProcess/CommandLineConfigurator.cs

```diff
@@ -91,6 +97,29 @@ private Option MakeOption<T>(OptionDefinition optionDefinition)
         return option;
     }
 
+    private async Task<bool> RunLink(string[] args, CancellationToken cancellationToken)
```

Comment:
Add comment explaining why this is here and why it couldn't be handled like other command line options.
This will prevent us from invertedly reverting this change due to not knowing the cause.
Or allow us to revert in if we happen to find a better solution to the issue, since cause is readily available.

Reply:
Also add this is only used on Linux